docs/tutorials/fundamentals/viewer.md

```diff
@@ -404,40 +379,53 @@ viewer.camera.zoom = 2
 viewer.camera.angles = (3, 38, 53)
 nbscreenshot(viewer, alt_text="A rotated 3D view")
 ```
+![image: Rotated image](../assets/tutorials/rotated-image.png)  
 
-Next to the 2D / 3D button is a button to roll the dimensions that are currently
-being displayed in the viewer - for example if you have a `ZYX` volume and are
-looking at the `YX` slice this will then show you the `ZY` slice.
+The third button rolls the dimensions that are currently displayed in the viewer. For example if you have a `ZYX` volume and are looking at the `YX` slice, this will then show you the `ZY` slice.
```

Comment:
```suggestion
The third button rolls the dimensions that are currently displayed in the viewer. For example if you have a `ZYX` volume and are looking at the `YX` slice, this will then show you the `ZY` slice. You can also right-click this button to re-order the dimensions by drag-and-drop.
```

Reply:
Bump. This isn't very discoverable, so i think it's very important to add here.